src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.module.css

```diff
@@ -1,45 +1,100 @@
 @media only screen and (max-width: 768px) {
-  .middleText {
+  .listItemTexts{
+    font-size: 16px;
+    --list-text-color-error: #E02E49;
+    flex: 1;
+    align-items: center;
+    margin-top: 8px;
+    margin-bottom: 8px;
+  }
+
+  .leftText {
+    flex: 1;
+    display: flex;
+    justify-content: flex-start;
+    word-break: break-words;
+
+  }
+
+  .middleText{
+    margin-top: 5px;
+    opacity: 60%;
+  }
+
+  .baseListItemContent{
+    display: flex;
+    flex: 1;
+    align-items: center;
+  }
+
+  .deleteSection {
+    display: flex;
     justify-content: flex-end;
+    align-items: center;
   }
-}
 
-.compactListItem {
-  display: flex;
-  margin-left: 10px;
-  min-height: 40px;
+  .compactListItem {
+    display: flex;
+    margin-left: 10px;
+    min-height: 40px;
+  }
 }
 
-.baseListItemContent{
-  display: flex;
-  flex: 1;
-  align-items: center;
-}
+@media only screen and (min-width: 769px) {
+  .leftText {
+    flex: 1;
+    display: flex;
+    justify-content: flex-start;
+    word-break: break-word;
+  }
 
-.listItemIcon {
-  margin-right: 10px;
+  .listItemTexts{
+    flex: 1;
+    display: flex;
+    font-size: 16px;
+    --list-text-color-error: #E02E49;
+    align-items: center;
+  }
+
+  .baseListItemContent{
```

Comment:
Looks like there's a lot of duplicate code between the two screen sizes, like this baseListItemContent for example, which seems to be the exact same in both cases.
Maybe only the code affected by the change in screen size should be defined separately for the two cases?